src/main/java/seedu/event/EventList.java

```diff
@@ -83,4 +83,14 @@ public void editVenue(int index, String venue) {
         Event event = this.find(index);
         event.setVenue(venue);
     }
+
+    public void editEvent(int index, Event event) {
+        list.remove(index);
```

Comment:
this might cause issues when the list does not contain this index

Reply:
thrown duke exception eventcommandinterpreter